SR2023/2022-09-29-confirm-team-details.md

```diff
@@ -0,0 +1,18 @@
+---
+to: SR2023 sign-ups
+subject: Confirming your details for SR2023
+---
+
+Hello!
+
+Thank you for your interest in Student Robotics 2023. We would like to offer you a place. In order to do that, we have a few questions and extra pieces of information we need.
+
+Please complete the following short form: https://forms.gle/TDhCmRpvhJHR4gWP9
+
+Once we have these details, we can confirm your place, and then you're in!
+
+While the details for kickstart are not yet available, the events are expected to take place on the same weekend in late October. We'll let you know as soon as those are available.
```

Comment:
Do we want to say it's the 22nd so at least they have a when even if not a where

Reply:
I think we should - it's getting quite close to that date now. If it's less than a couple of weeks away we'd probably be best off postponing to give more prep time

Reply:
```suggestion
While the details for kickstart are not yet available, the events are expected to take place on the 22nd October. We'll let you know as soon as those are available.
```

Reply:
Assuming we actually have a booking for that date, that text looks good.

Reply:
I don't believe we do quite yet.

Reply:
```suggestion
While the details for kickstart are not yet available, the events will provisionally be on the 22nd October and be in London and Southampton. We'll let you know as soon as those are available.
```